tests/integration/records/test_records_json_ref_loader.py

```diff
@@ -29,7 +29,7 @@
 def test_load_resolved_schema(app):
     unresolved_schema = load_schema('hep')
 
-    assert '$ref' in unresolved_schema['properties']['urls']['items']
+    assert '$ref' not in unresolved_schema['properties']['urls']['items']
```

Comment:
I would remove the `assert`, as the behavior we really care about is that at the end schemas are resolved.

Reply:
done